tests/integration/security/rbac/testdata/istio-rbac-v2-rules.yaml.tmpl

```diff
@@ -0,0 +1,129 @@
+# istio-rbac-v2-rules.yaml to enforce access control for both http and tcp services using Istio RBAC v2 rules.
```

Comment:
We will need a test case for namespace authz policy, which means the workload_selector is not specified and there is also no destination.labels in ServiceRole. We can do this in follow-up PRs.

Reply:
Yes. We can add more tests in follow-up PRs. This is just to get the basic tests in.